docs/05-infrastructure-nodes.md

```diff
@@ -0,0 +1,318 @@
+# OpenShift Infrastructure Nodes
+The OpenShift subscription model allows customers to run various core
+infrastructure components at no additional charge. In other words, a node
+that is only running core OpenShift infrastructure components is not counted
+in terms of the total number of subscriptions required to cover the
```

Comment:
hi, can you please add more info about _not counted
in terms of the total number of subscriptions required_ on how that works? 

And while i see this is very much focused on OCP (fair point), i'm sure a lot of folks from OKD world will come across this docs (very useful ones, don't get me wrong !) and they will panic on the above statement tbh.

Reply:
Hey there --

* https://www.okd.io/ refers to itself as the "community distribution of kubernetes that powers openshift". OKD itself is not OpenShift. OKD has no subscription model. I'm not understanding why you think someone would panic about a discussion on a subscription model for enterprise software?

* This is only focused on OCP. Derek's original repo is the "generic" version.

* I'm not sure what "more" info you are looking for? I don't think there's much more to add. I'm happy to take suggestions!

Reply:
> OKD itself is not OpenShift

eh ?! that is an interesting statement ... If that is the case then i guess we can close any further discussion. thx for taking the time to respond.